repository/kafka/docs/v1.0/versions.md

```diff
@@ -0,0 +1,10 @@
+### Versions
+
+- Apache Kafka 2.12-2.3.0
+- OpenJDK 8
+- JMX Exporter 0.1.0
+
+
+### KUDO Version Requirement
+
+KUDO version 0.4.0 or later
```

Comment:
Should this be changed to 0.8.0?

Reply:
I'm pretty sure 0.8.0, but it may be 1.0.0 - TBD until tomorrow.

Reply:
let me update 0.8.0 everywhere and we can update this PR later also